File: ScrumChores.Model/Entities/Story.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ScrumChores.Model.Entities
{
    public class Story
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public Guid StoryID { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public int Effort { get; set; }
        public Sprint Sprint { get; set; }
        public virtual List<SprintTask> Tasks { get; set; } 
    }
}

File: ScrumChoresPublicKO/Scripts/ScrumChores/Common/Models/Story.js
scrumChores.models.story = (function (Sprint, StoryID, Title, Description, Effort) {
    var self = this;
    self.Sprint = ko.observable(Sprint);
    self.StoryID = ko.observable(StoryID);
    self.Title = ko.observable(Title);
    self.Description = ko.observable(Description);
    self.Effort = ko.observable(Effort);
});
         